modules/v2x/proto/v2x_monitor.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto2";
+
+package apollo.obu.v2x_monitor;
+
+enum ErrorCode {
+    LTEV = 500;
+    NET = 501;
+    CPU = 502;
+    MEM = 503;
+    GPS = 504;
+    MAP = 510;
+    SPAT = 511;
+    OBUID = 999;
+}
```

Comment:
What does `LTEV` `SPAT`, `OBU` mean?

Reply:
https://en.wikipedia.org/wiki/Vehicle-to-everything